helper/linux/writejob.h

```diff
@@ -63,4 +66,21 @@ public slots:
     QProcess *dd { nullptr };
 };
 
+
+template<class T>
+static std::tuple<std::unique_ptr<T[]>, T*, std::size_t> pageAlignedBuffer(std::size_t pages = 1024) {
```

Comment:
So since this is the first template implementation in this project I wasn't sure where to put the implementation.
Maybe this should be moved into some sort of utility helper file too.